docs/public/s2i-to-devfile.adoc

```diff
@@ -0,0 +1,92 @@
+== Migrating from S2I components to Devfile components
+
+Devfiles are new way of deploying component with odo that offers more convenience and flexibility. Read more about devfiles https://devfile.github.io/[here]
```

Comment:
This seems opinionated. I think we should just say that it's a new deployment method. Saying that it's more convenience or flexible isn't necessarily true. 

Instead of adding "Read more about Devfiles" you can instead just make the first word of the sentence a clickable link.